styles/header.css

```diff
@@ -75,3 +112,16 @@
     }
   }
 }
+
+.navbar .dropdown-menu {
+  --bs-dropdown-padding-x: 0.125rem;
+  --bs-dropdown-padding-y: 0.125rem;
+  --bs-dropdown-item-padding-x: 0.5rem;
+  --bs-dropdown-item-padding-y: 0.125rem;
+
+  &.show {
+    display: flex;
```

Comment:
I believe this is causing the content of the menu to run off the edge of the screen, which makes it impossible to read.:
<img width="288" alt="Screenshot 2025-01-23 at 12 31 26 PM" src="https://github.com/user-attachments/assets/63776aa9-6f3c-4bd0-a821-317192f6004e" />

Reply:
It seems like popper isn't working correctly.

Reply:
Possibly due to https://github.com/twbs/bootstrap/blob/5708adc816b9a6b07fb2b4f07c3b562935bd0a54/site/content/docs/5.3/migration.md?plain=1#L603C3-L604C1

Reply:
Perhaps we don't want to use `.navbar` here.

Reply:
Removing this `&.show` block doesn't resolve that issue for me.

Reply:
Removing `.navbar` does seem to, though...I'll see if I can work around that

Reply:
Added a version without `.navbar`. Popper now positions things correctly, but the tradeoff is that the dropdowns no longer take up space in the header on when opened on mobile and float above it instead, which is different from the designs.

Reply:
Resolved in favor of this as per slack discussion.